<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVImageFilterClass</class>
 <widget class="QMainWindow" name="OpenCVImageFilterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCVImageFilter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_openLoadPath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>openLoadPath</string>
        </property>
        <property name="icon">
         <iconset resource="opencvimagefilter.qrc">
          <normaloff>:/OpenCVImageFilter/Resources/open.png</normaloff>:/OpenCVImageFilter/Resources/open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_loadFilePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>../1.jpg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_openSavePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>openSavePath</string>
        </property>
        <property name="icon">
         <iconset resource="opencvimagefilter.qrc">
          <normaloff>:/OpenCVImageFilter/Resources/open.png</normaloff>:/OpenCVImageFilter/Resources/open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_saveFilePath">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>../2.jpg</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="cmb_imageEditSln">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>-----------Please select an image editing----------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>autoCutAll</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cutImage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rotateImage</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>highGaussianFilter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>highButterworthFilter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lowGaussianFilter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>lowButterworthFilter</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>filterRadius</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="edt_filterRadius"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ButterworthNum</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="edt_ButterworthNum"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>lineNum</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="edt_lineNum">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Please load an image.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>image</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="actionEditImage"/>
    <addaction name="actionSaveImage"/>
   </widget>
   <addaction name="menuImage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadImage"/>
   <addaction name="actionEditImage"/>
   <addaction name="actionSaveImage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadImage">
   <property name="icon">
    <iconset resource="opencvimagefilter.qrc">
     <normaloff>:/OpenCVImageFilter/Resources/save3.png</normaloff>:/OpenCVImageFilter/Resources/save3.png</iconset>
   </property>
   <property name="text">
    <string>loadImage</string>
   </property>
  </action>
  <action name="actionEditImage">
   <property name="icon">
    <iconset resource="opencvimagefilter.qrc">
     <normaloff>:/OpenCVImageFilter/Resources/rst.png</normaloff>:/OpenCVImageFilter/Resources/rst.png</iconset>
   </property>
   <property name="text">
    <string>editImage</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="icon">
    <iconset resource="opencvimagefilter.qrc">
     <normaloff>:/OpenCVImageFilter/Resources/save.png</normaloff>:/OpenCVImageFilter/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>saveImage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="opencvimagefilter.qrc"/>
  <include location="../../OpenCVDemo(ver1.0.1)/OpenCVDemo/opencvdemo.qrc"/>
 </resources>
 <connections/>
</ui>
